app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -217,5 +222,22 @@ def action_permission
         fail ::Foreman::Exception.new(N_("unknown permission for %s"), "#{params[:controller]}##{params[:action]}")
       end
     end
+
+    def find_content_overrides
+      if params[:content_overrides_search]
+        content_labels = ::Katello::Content.joins(:product_contents).
+                            where("#{Katello::ProductContent.table_name}.product_id": @host.organization.products.subscribable.enabled).
+                            search_for(params[:content_overrides_search][:search]).
+                            pluck(:label)
```

Comment:
Could I please have my dots on the next line?  (Isn't there a cop for this?) :see_no_evil:

Reply:
not a fan of dot on the next line?  Do you see it being done that way elsewhere?

Reply:
https://github.com/jeremylenz/katello/blob/d4f2976f4d28e574d291dcbb793afa02b7b8a363/app/controllers/katello/api/v2/repositories_controller.rb#L203

https://github.com/jeremylenz/katello/blob/d4f2976f4d28e574d291dcbb793afa02b7b8a363/app/controllers/katello/api/v2/repositories_controller.rb#L208

https://github.com/jeremylenz/katello/blob/d4f2976f4d28e574d291dcbb793afa02b7b8a363/app/controllers/katello/api/v2/repositories_controller.rb#L198

https://github.com/jeremylenz/katello/blob/d4f2976f4d28e574d291dcbb793afa02b7b8a363/app/controllers/katello/api/v2/repositories_controller.rb#L193

https://github.com/jeremylenz/katello/blob/304de977b189990164968a7d1a482137012bc9d7/test/controllers/api/registry/registry_proxies_controller_test.rb#L136

https://github.com/jeremylenz/katello/blob/304de977b189990164968a7d1a482137012bc9d7/test/controllers/api/registry/registry_proxies_controller_test.rb#L137

https://github.com/jeremylenz/katello/blob/13cd0c84879a430daa052f4dc9ffdff20cc1fb8b/test/models/ping_test.rb#L94

https://github.com/jeremylenz/katello/blob/13cd0c84879a430daa052f4dc9ffdff20cc1fb8b/test/models/ping_test.rb#L94

let me know if you need more :wink:

Reply:
I guess I'm used to JS.. eslint wouldn't let you do it the other way